src/main/java/picard/fingerprint/CheckFingerprint.java

```diff
@@ -217,12 +252,25 @@ else if (!SAMPLE_ALIAS.equals(rec.getSample())) {
             }
 
             summaryFile.addMetric(metrics);
-            log.info(metrics.READ_GROUP + " vs. " + metrics.SAMPLE + ": LOD = " + metrics.LOD_EXPECTED_SAMPLE);
+            log.info("Read Group: " + metrics.READ_GROUP + " / " + metrics.OBSERVED_SAMPLE_ALIAS + " vs. " + metrics.SAMPLE + ": LOD = " + metrics.LOD_EXPECTED_SAMPLE);
         }
 
         summaryFile.write(outputSummaryMetricsFile);
         detailsFile.write(outputDetailMetricsFile);
 
         return 0;
     }
+
+    protected String[] customCommandLineValidation() {
+        IOUtil.assertFileIsReadable(INPUT);
```

Comment:
I put a copy back where it was.  Needed to validate readability here since this is done before doWork is hit.